maquetado/scss/_tech.scss

```diff
@@ -0,0 +1,68 @@
+.content {
+  margin: 30px auto;
+  max-width: 100%;
+  width: 1024px;
+}
+.title {
+  width: 100%;
+}
+.section-tech {
+  display: flex;
+  flex-direction: row;
+  flex-flow: wrap;
+  justify-content: space-between;
```

Comment:
I think that this isn't necessary, delete it

Reply:
I didn't delete this line because is necessary for the space between cards of tecnologia